src/main/java/minicraft/level/tile/farming/PlantTile.java

```diff
@@ -19,8 +20,13 @@ protected PlantTile(String name) {
 
     @Override
     public void steppedOn(Level level, int xt, int yt, Entity entity) {
-        super.steppedOn(level, xt, yt, entity);
+		if (entity instanceof ItemEntity) return;
+		// Very low chance because steppedOn gets called every tick of movement on the tile.
+		if (random.nextInt(200) != 0) return;
```

Comment:
I think the chance is too low compared to the chance changing back into a dirt tile.

Reply:
Okay, what do you think it should be? Keep in mind that this is called every tick that you are moving on the tile, with 60 ticks per second.

Reply:
It is not every tick. If a player is passing through a farmland straight, this is called only 16 times in total with normal speed.

Reply:
I just tested it with `Logging.TILES.debug("stepping" + System.currentTimeMillis());` and after a second theres 60 lines with every datapoint having ~16ms delay from each other though... I had a feeling of this because on `FarmTile.steppedOn` it's 1/60 chance and it took around 1 second to turn the `FarmTile` to dirt.

Reply:
> I think the chance is too low compared to the chance changing back into a dirt tile.

I will change it to 100 and 10 respectively.